### PR TITLE
[zh-cn]: update the translation of Number.MAX_SAFE_INTEGER

### PR DESCRIPTION
**Pull request by JasonLamv-t**
### Description

Same as the title.

### Motivation

Ditto.

### Additional details

None.

### Related issues and pull requests

#15736

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/number/max_safe_integer/index.md [hunk 1]
```diff
@@ -7,21 +7,36 @@ slug: Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER
...
+### MAX_SAFE_INTEGER 的返回值
+
 ```js
 Number.MAX_SAFE_INTEGER; // 9007199254740991
-Math.pow(2, 53) - 1; // 9007199254740991
+```
+
+### MAX_SAFE_INTEGER 和 EPSILON 之间的关系
+
+{{jsxref("Number.EPSILON")}} 是 2<sup>-52</sup>，而 `MAX_SAFE_INTEGER` 是 2<sup>53</sup> – 1，它们都源自尾数的宽度，即 53位（最高位始终为 1）。将它们相乘会得到一个非常接近但不等于 2 的值。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/number/max_safe_integer/index.md:36]:
```suggestion
{{jsxref("Number.EPSILON")}} 是 2<sup>-52</sup>，而 `MAX_SAFE_INTEGER` 是 2<sup>53</sup> – 1，它们都源自尾数的宽度，即 53 位（最高位始终为 1）。将它们相乘会得到一个非常接近但不等于 2 的值。
```

File: files/zh-cn/web/javascript/reference/global_objects/number/max_safe_integer/index.md [hunk 2]
```diff
@@ -7,21 +7,36 @@ slug: Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER
 
 **`Number.MAX_SAFE_INTEGER`** 静态数据属性表示在 JavaScript 中最大的安全整数（2<sup>53</sup> – 1）。
 
+对于更大的整数，请考虑使用 {{jsxref("BigInt")}}。
+
+{{EmbedInteractiveExample("pages/js/number-maxsafeinteger.html")}}
+
+## 值
+
+`9007199254740991` (9,007,199,254,740,991，或者 \~9 千万亿).
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/number/max_safe_integer/index.md:16]:
```suggestion
`9007199254740991`（9,007,199,254,740,991，或者约 9 千万亿）。
```

File: files/zh-cn/web/javascript/reference/global_objects/number/max_safe_integer/index.md [hunk 3]
```diff
@@ -32,7 +47,9 @@ Math.pow(2, 53) - 1; // 9007199254740991
 
 {{Compat}}
 
-## 相关链接
+## 参见
 
+- [Polyfill of `Number.MAX_SAFE_INTEGER` in `core-js`](https://github.com/zloirock/core-js#ecmascript-number)
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/number/max_safe_integer/index.md:52]:
```suggestion
- [`core-js` 中 `Number.MAX_SAFE_INTEGER` 的 polyfill](https://github.com/zloirock/core-js#ecmascript-number)
```

**Review by yin1999 — APPROVED** [review]
Thank you @JasonLamv-t

**Pull request merged**